custom_addons/incident_management/models/x_inc_investigation.py

```diff
@@ -23,7 +24,10 @@ def create(self, vals_list):
         if incident_id:
             incident = self.env['x.incident.record'].browse(incident_id)
             incident.write({'state': 'investigation_assigned'})
-        investigation.action_send_investigation_email()
+            investigation.action_send_investigation_email()
```

Comment:
Why this line has gone inside the if condition
investigation.action_send_investigation_email()

Only this line should be inside if condition and  both the lines wrt to investigation is outside if condition